dts/arm/nxp/nxp_rt6xx_common.dtsi

```diff
@@ -60,9 +60,14 @@
 
 			status = "disabled";
 
-			region@24000000 {
+			/*region@24000000 {
 				reg = <0x24000000 DT_SIZE_K(64)>;
 				type = <NXP_RTXXX_ADSP_REGION_DATA>;
+			};*/
+
```

Comment:
We shouldn't have commented code in the final version. So you should remove this. One can look at the history and see the changes.